### DISCUSSION TITLE
How do I manage what drive fooocus will work on?

### DISCUSSION THREAD

**Discussion by vincent-bru** [lllyasviel/Fooocus#3583] [Q&A]
Windows 10
When I use fooocus, after few generation I got my C: drive full and a windows alert. I have 16g remaining on it so I understand it's not enough, I'd like to go with my other drive which have 100g free. I have seen the "System swap" solution, but I only want fooocus to do that, I don't want my other soft to work on secondary drive because of that

**Comment by Aerthlyomi** [comment]
On the main page of Fooocus, (https://github.com/lllyasviel/Fooocus) look for the section Customization. It will give you examples of elements you can change in the config.txt 
The one you are looking for is  "path_outputs": "D:\\Fooocus\\outputs",
Change the Drive letter to reflect your own configuration and respect the syntax.

Do not worry of doing a mistake, if Fooocus doesn't restart, just delete the config.txt  Fooocus will recreate one by default at the next start.

**Reply by vincent-bru** [reply]
It's not that. The output is not the problem. The problem is caching. No matter what drive I install on or I put in the config file, the system drive get bursted by datas during runtime

**Reply by Rogala** [reply]
In Windows OS there is only one swap file and the system uses it at its discretion for all programmes that run.

The swap file for Focus is recommended 40 GB, and the OS is recommended to have another 15% of free space without taking into account this 40 GB.
